src/components/Nav/Nav.js

```diff
@@ -1,7 +1,231 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled, { css } from 'styled-components';
+import { FaSistrix } from 'react-icons/fa';
 
 const Nav = () => {
-  return <div></div>;
+  const [category, setCategory] = useState(0);
+  return (
+    <NavBar>
+      <NavHeader>
+        <HeaderBox>
+          <Navimg alt="ourtriplogo" src="/images/NavLogo.png" />
+          <HeaderSearch>
+            <SearchContainer>
+              <SearchInput>
+                <TripIcon size="20" color="#676D74" />
+                <input type="text " placeholder="도시나 상품을 검색해주세요" />
+              </SearchInput>
+            </SearchContainer>
+          </HeaderSearch>
+          <HeaderItems>
+            <NavItems>
+              <span>파트너 등록하기</span>
+              <span>로그인</span>
+              <span>회원가입</span>
+            </NavItems>
+          </HeaderItems>
+        </HeaderBox>
+      </NavHeader>
+      <NavBarSubCategory>
+        <NavBarSub>
+          <NavList>
+            {CATEGORY_LIST.map(item => (
+              <SubList
+                key={item.id}
+                onClick={() => setCategory(item.id)}
+                active={category === item.id}
+              >
+                {item.title}
```

Comment:
콜백함수의 인자부분에서 구조분해 할당이 가능해서 데이터가 많지 않다면, 구조분해 할당해서 더 깔끔하게 할 수 있습니다.
```suggestion
  {CATEGORY_LIST.map(({ id, title }) => (
              <SubList
                key={id}
                onClick={() => setCategory(id)}
                active={category === id}
              >
                {title}
              </SubList>
            ))}
```